src/Settings/CSS/stylesheet.css

```diff
@@ -282,11 +282,7 @@ html.two-columns #illustration-buffer {
 
 .restart:hover {
   background-color: var(--background-secondary);
```

Comment:
So, I tested this on dark mode, and it's not great. Might wanna remove this.

Reply:
Do you mean to remove the background-color? This has not been changed and is as it exists in the live version. I can remove if desired just wanted to check first!